mozilla-extensions/bergamot-browser-extension
# Release v0.4.1

This PR bumps the version, creating a commit that will be tagged with the v0.4.1 tag after merge.

# Suggested release notes:

## TL;DR:

This release contains bug fixes and smaller improvements.

Installation instructions are found [here](https://github.com/mozilla-extensions/bergamot-browser-extension/blob/v0.4.1/docs/INSTALL.md). 

### Changes compared to v0.4.0:
- Reduce bundle size #209
- Telemetry-related:
  - Change cpu family, model and stepping in telemetry to quantity #223
  - Upgrade Glean.js to v0.15.0 #215
  - Avoid a potential short-lived incorrect document translation state when a new translation is requested #185
  - Submit telemetry ping on every new unsupported language detected #184
  - Add word counts to telemetry #183
- Test-related:
  - Make source maps usable in production builds #182
- Dev-related:
  - Update fixtures url in two places where it should be updated #217 
  - Remove the taskcluster integration #207
  - Clarify which dependencies are used by which UI variant #174

For more information, visit [the main readme](https://github.com/mozilla-extensions/bergamot-browser-extension/tree/v0.4.1) which contains links to installation instructions and documentation for developers.
